scrapper/main.py

```
@@ -0,0 +1,109 @@
+        "version": "1.0.0",
+        "status": "running",
+        "endpoints": {
+            "health": "GET /health",
+            "ingest": "POST /internal/ingest",
+            "docs": "GET /docs"
+        }
+    }
+
+
+@app.exception_handler(Exception)
+async def global_exception_handler(request, exc):
```

Ruff (0.15.12)

[warning] Unused function argument: `request`

(ARG001)